src/javaRosa.js

```diff
@@ -44,7 +44,8 @@ define([
             'default', 'short', 'long', 'audio', 'video', 'image'
         ],
         _nextItextItemKey = 1,
-        NO_MARKDOWN_MUGS = ['Item', 'Group', 'FieldList', 'Repeat'];
+        NO_MARKDOWN_MUGS = ['Item', 'Group', 'FieldList', 'Repeat'],
+        EXPERIMENTAL_UI;
```

Comment:
Would it be hard to move all these `EXPERIMENTAL_UI` checks into `widgets.multilineText`? the implication is that every place that currently uses `multilineText` would be switched to `richText` when experimental UI is on.